geode-core/src/distributedTest/java/org/apache/geode/internal/cache/RemoteTransactionDUnitTest.java

```diff
@@ -3666,27 +3570,24 @@ public boolean getSuccess() {
 
     @Override
     public void afterCreate(EntryEvent event) {
-      fail("create not expected");
+      throw new UnsupportedOperationException("create not expected");
     }
 
     @Override
     public void afterUpdate(EntryEvent event) {
-      if (!success) {
-        System.out.println("WE WIN!");
-        success = true;
-      } else {
-        fail("Should have only had one update");
-      }
+      assertThat(success).as("Should have only had one update").isTrue();
```

Comment:
To preserve the original behaviour, this should be `assertThat(success).as("Should have only had one update").isFalse();`